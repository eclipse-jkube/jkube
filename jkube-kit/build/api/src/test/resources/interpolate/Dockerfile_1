FROM ${base}
LABEL name ${name} @age@ ${ext} ${unknown} ${project.artifactId}
ENV ${name} ${age}
ENV cli=${cliOverride}
USER ${user.name}