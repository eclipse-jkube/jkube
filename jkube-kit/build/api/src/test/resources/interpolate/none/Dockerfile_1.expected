FROM @base@
LABEL name ${name} @age@ ${ext} blub
ENV @name@ @age@
